runtime/development/src/lib.rs

```diff
@@ -1907,9 +1907,94 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
-	pallet_block_rewards::migrations::InitBlockRewards<Runtime, CollatorRewards, TotalRewards>,
+	UpgradeDev1020,
 >;
 
+type UpgradeDev1020 =
+	pallet_block_rewards::migrations::InitBlockRewards<Runtime, CollatorRewards, TotalRewards>;
```

Comment:
Why is this migration needed for development?

Reply:
For the `demo` chain and to enable testing the migration locally before we add it to `Altair` and `Centrifuge` later.

Reply:
Ok, so as a norm, we should always create migrations for any breaking change in `development,` right?

Reply:
I would say so, yes.